### DISCUSSION TITLE
C#代码高亮[Feature Request]

### DISCUSSION THREAD

**Discussion by aiyuechuang** [vuepress-theme-hope/vuepress-theme-hope#3149] [Q&A] [answered]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

C#代码高亮
<img width="854" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/61228664/5ed84708-e240-4d9a-ac0f-990b77a2857e">

**Comment by Mister-Hope** [comment, marked as answer]
```cs
using ABC;
```

```csharp
using ABC;
```

````md
```cs
using ABC;
```

```csharp
using ABC;
```
````

https://prismjs.com/#supported-languages

![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/987b1a39-fd86-447d-8df7-5ced021d98dc)


## ~~If you do think that `c#` fence should be supported, post an issue to prismjs.~~

## Edited: https://github.com/PrismJS/prism/issues/3681
